pkg/model/logstore/prefix.go

```diff
@@ -11,12 +11,12 @@ func SourcePrefix(n model.ManifestName) string {
 	if n == "" || n == model.TiltfileManifestName {
 		return ""
 	}
-	max := 12
+	max := 13
```

Comment:
how hard would it be to move this change to a separate PR? it affects a lot of tests, and doesn't seem related to the rest of the change.

Reply:
Sure, I can make this a separate PR!

Reply:
Nevermind — discussed this elsewhere with Nick and decided it was too much trouble.

(Many of our tests for logs just test expected hard-coded strings against actual output. We should put these into our snapshot test system so Golden Master for everything can be generated with one command.)